cmd/generatePlayerStruct.go

```diff
@@ -51,12 +53,18 @@ func New%s(x, y float64, file string) (*%s, error) {
         return nil, fmt.Errorf("failed to parse aseprite file: %%w", err)
     }
 
+    audioHandler, err := audio.InitAudioHandler("fantasy-8bit.mp3", audio.MP3)
```

Comment:
what's that `fantasy-8bit-mp3` ? Should we include it as part of the generation?